_episodes/02-software-development.md

```diff
@@ -237,16 +226,73 @@ depicts the use of MVC architecture for the [DNA Guide Graphical User Interface
 > {: .solution}
 {: .challenge}
 
+> ## Separation of Concerns
+> Separation of concerns is important when designing software architectures in order to reduce the code's complexity.
+> Note, however, there are limits to everything - and MVC architecture is no exception. Controller often transcends
+> into Model and View and a clear separation is sometimes difficult to maintain. For example, Command Line Interface
+> provides both the View (what user sees and how they interact with the shell) and the Controller (invoking of a command)
+> aspects of a CLI application. In Web applications, Controller often manipulates the data (received from the Model)
+> before displaying it to the user or passing it from the user to the Model.
+>
+{: .callout}
+
 #### Our Project's MVC Architecture 
 
-Our software project uses the MVC architecture. The file `patientdb.py` is the Controller module that performs 
+Our software project uses the MVC architecture. The file `patientdb.py` is the **Controller** module that performs 
 basic statistical analysis over data and provides the main
-entry point into the application. The View and Model modules are contained 
-in the files `view.py` and `model.py`, respectively, and are conveniently named. Data underlying the Model is contained within
+entry point into the application through the function called `main()` (as you can see from its listing below). 
+
```

Comment:
It might be confusing to put the code here without explaining a bit more about it?

Reply:
Do you think just not mentioning the main() function and leaving out the code listing would be better here? E.g. just say "The file `inflammation-analysis.py` is the Controller module that performs basic statistical analysis over data and provides the main entry point into the application" without much further explanation as to how and why it provides an entry point. @steve-crouch

Reply:
Or shall I add an explanation of/link to an explanation of the main() function - in which case some code snippet may be useful?

Reply:
Yes, I was unsure of this myself. Leaving it out should be fine, since as the section says, how the patient system fits within MVC is well explained in the Software Design section, and they don't need to know that in detail until then. What do you think @anenadic ?

Reply:
Sounds good to me @steve-crouch - will remove the code listing and mentioning the main() functions and they will have to believe us at this point that `inflammation-analysis.py` is the entry point into the application.